.nvmrc

```diff
@@ -1 +1 @@
-v10.16.3
+v12.18.2
```

Comment:
How come this needs a version bump?

web is still running 10.x IIRC so we still need to support that

Reply:
Because of `Object.fromEntries`. Same reason why TS needs ES2019.

How come web still uses node 10? 😁 Damn... 😅

Reply:
I believe that'd be a question for @donnielrt. But the answer is probably something like "breaking changes in 12.x and folks not yet having the time to diagnose and address them".

End result, I think we still need to keep 10.x support for the time being, and the easiest way to do that is keep the tests running in 10.x, so we probably want to avoid fromEntries for the time being :(

(even though a version bump would [save me some headaches too](https://github.com/Shopify/quilt/pull/1454))

Reply:
Yeah, anyway I wanted to revert the version anyway following your first message since it wouldn't be wise to bump it here if web is not able to easily/quickly do it as well. Having said that, I still think the web (products) teams should spend some time to try and see if their code works in node 12. Node 14 LTS is just a few months away...

Reply:
> the answer is probably something like "breaking changes in 12.x

Yeah, it was mostly 100s of section tests that were failing, we've embarked on a big effort to fix and modernize tests, which should clear the path for 12.x 🤞

Reply:
PR in question: https://github.com/Shopify/web/pull/28941. The Supersonic Web project is making a bunch of improvements to the dev experience, we might look at getting in Node 12 as part of that.